test_leepfrog.py

```diff
@@ -0,0 +1,5 @@
+fginwfnf
+gwnigwgnw
+def fegje() 
```

Comment:
This function declared in a wrong way. 
```suggestion
def fegje():
  # DO SMTHNG
  return 0
```